converters/fromZigbee.js

```diff
@@ -7338,13 +7338,14 @@ const converters = {
             const dpValue = tuya.firstDpValue(msg, meta, 'SLUXZB');
             const dp = dpValue.dp;
             const value = tuya.getDataValue(dpValue);
+            const brightnesStateLookup = {'0': 'LOW', '1': 'MEDIUM', '2': 'HIGH'};
             switch (dp) {
             case 2:
                 return {illuminance_lux: value};
             case 4:
                 return {battery: value};
             case 1:
-                return {battery_low: value === 1};
+                return {brightness_state: brightnesStateLookup[value]};
```

Comment:
`brightness_level` sounds like a better name to me

Reply:
Thanks, I changed this.